cmd/kubelet/app/server.go

```diff
@@ -474,6 +476,30 @@ func makeEventRecorder(kubeDeps *kubelet.Dependencies, nodeName types.NodeName)
 	}
 }
 
+func getReservedCPUs(machineInfo *cadvisorapi.MachineInfo, cpus string) cpuset.CPUSet {
+	emptyCPUSet := cpuset.NewCPUSet()
+	if cpus != "" {
+		topo, err := topology.Discover(machineInfo)
+		if err != nil {
+			klog.Error("Unable to discover CPU topology info")
```

Comment:
Maybe embed the error into this logline to give some context?

Reply:
ack